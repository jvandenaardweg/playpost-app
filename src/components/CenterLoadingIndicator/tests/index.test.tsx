import React from 'react';
import { render, RenderAPI } from 'react-native-testing-library';

import { CenterLoadingIndicator } from '../index';

describe('CenterLoadingIndicator', () => {

  describe('rendering', () => {
    let wrapper: RenderAPI;

    beforeAll(() => {
      wrapper = render(<CenterLoadingIndicator />);
    });

    it('should render correctly', () => {
      expect(wrapper.toJSON()).toMatchSnapshot();
    });
  });
});
