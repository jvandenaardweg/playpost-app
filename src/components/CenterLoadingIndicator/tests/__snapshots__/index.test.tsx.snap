// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenterLoadingIndicator rendering should render correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#EEEEEE",
        "bottom": 0,
        "flex": 1,
        "height": "100%",
        "justifyContent": "center",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "width": "100%",
      },
      Object {
        "backgroundColor": undefined,
      },
    ]
  }
>
  <ActivityIndicator
    animating={true}
    color="#040404"
    hidesWhenStopped={true}
    size="small"
  />
</View>
`;
